Dockerfile

```diff
@@ -59,6 +60,16 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" &&
 php composer-setup.php --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && \
 php -r "unlink('composer-setup.php');"
 
+### Travis
+# Install travis cli and ruby dependencies
+#   Also installs dev dependencies (build_deps) for compiling ruby-ffi
+#   which are removed after install
+RUN apk add --no-cache ruby ruby-io-console ruby-json git && \
+    apk add --no-cache --virtual build_deps \
+    build-base ruby-dev libc-dev libffi-dev linux-headers && \
+    gem install travis -v ${TRAVIS_CI_CLI_VERSION} --no-rdoc --no-ri && \
+    apk del build_deps
```

Comment:
Maybe we should run cleanup here as well? Not sure how needed it is, but it would be consistent with other `apk add` commands we have in our images:
```
# Cleanup
    rm -rf /tmp/* && \
    rm -rf /var/cache/apk/*
```